hack/lib/init.sh

```diff
@@ -6,17 +6,17 @@ ROOT_PATH=$(dirname "${BASH_SOURCE[0]}")/../..
 
 declare -A CODE_PATH=(
 	["observer-default-plugins"]="observer-plugins/default-plugins"
-	["executor-resource-tagger"]="executor-plugins/default-plugins"
+	["executor-default-plugins"]="executor-plugins/default-plugins"
 )
 
 declare -A CODE_MAIN_PATH=(
 	["observer-default-plugins"]="main.go"
```

Comment:
Because of changing `observer-plugins/default-plugins/main.go` -> `observer-plugins/default-plugins/cmd/server/server.go`.

So this line should change to `cmd/server/server.go`

Reply:
fixed, missed this one